app/models/experiment.rb

```diff
@@ -716,6 +713,16 @@ def generate_parameter_values(parameter)
     parameter_values
   end
 
+  def evaluate_given_input_parameter_value(name_of_input_parameter, value_of_input)
```

Comment:
Nazwa metody: to co robi ta metoda to rzucanie wyjątkiem, jeśli wartość parametru jest niepoprawnego formatu. Może lepszą nazwą byłoby: ``validate_parameter_value`` ?

Na przyszłość: słowo ``given`` w nazwie metody jest w takich przypadkach nadmiarowe.